package.json

```diff
@@ -82,7 +82,7 @@
     "babel-loader": "^8.0.2",
     "electron": "4.1.1",
     "electron-builder": "^20.39.0",
-    "electron-webpack": "^2.0.1",
+    "electron-webpack": "^2.6.2",
```

Comment:
Can you drop this change? Upgrading electron-webpack breaks our prod builds (see #365).